azure_jumpstart_ag/artifacts/PowerShell/AgLogonScript.ps1

```diff
@@ -174,12 +174,11 @@ if ($industry -eq "retail") {
 }
 
 if ($industry -eq "manufacturing") {
-    #Deploy-ESA -AgConfig $AgConfig -Credentials $Credentials
     Deploy-AIO
+    Deploy-ManufacturingConfigs -AgConfig $AgConfig -Credentials $Credentials
```

Comment:
I saw that they are available globally but that means our functions are dependent on those globals rather than capable of being used elsewhere with the credentials and hashtable objects. I know its not perfect that way either since its a dependency on those objects.

Reply:
I'm fine both ways, either way we will have to standardize the selected method on all functions.